CMakeLists.txt

```diff
@@ -856,9 +856,9 @@ if (HDF5_ENABLE_SUBFILING_VFD)
 endif()
 
 
-#option (DEFAULT_API_VERSION "Enable v1.16 API (v16, v18, v110, v112, v114, v116)" "v116")
-set (DEFAULT_API_VERSION "v116" CACHE STRING "Enable v1.16 API (v16, v18, v110, v112, v114, v116)")
-set_property (CACHE DEFAULT_API_VERSION PROPERTY STRINGS v16 v18 v110 v112 v114 v116)
+#option (DEFAULT_API_VERSION "Enable v1.16 API (v16, v18, v110, v112, v114, v116, v118)" "v116")
+set (DEFAULT_API_VERSION "v116" CACHE STRING "Enable v1.16 API (v16, v18, v110, v112, v114, v116, v118)")
```

Comment:
Should update `DEFAULT_API_VERSION` to v118 and probably removed the commented out line above

Reply:
Agree